tokio/CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+# 1.10.0 (August 12, 2021)
+
+### Added
+
+ - io: add `(read|write)_f(32|64)[_le]` methods ([#4022])
+ - io: add `fill_buf` and `consume` to `AsyncBufReadExt` ([#3991])
+ - process: add `Child::raw_handle()` on windows ([#3998])
+
+### Fixed
+
+ - doc: fix non-doc builds with `--cfg docsrs` ([#4020])
```

Comment:
IIUC the problem that this PR fixed does not exist in the released version.
```suggestion
```

Reply:
The change in `tokio/src/macros/cfg.rs` did actually also fix something in the released version See the `#tokio-docs` channel in the discord.

Reply:
I added an explanation of this in the relevant PR description.